src/Illuminate/Collections/Collection.php

```diff
@@ -805,6 +805,17 @@ public function mergeRecursive($items)
         return new static(array_merge_recursive($this->items, $this->getArrayableItems($items)));
     }
 
+    /**
+     * Merge the given items with the collection.
+     *
+     * @param  \Illuminate\Contracts\Support\Arrayable<TKey, TValue>|iterable<TKey, TValue>  $items
+     * @return static
+     */
+    public function mergeUnshift($items)
```

Comment:
Can you add tests to types here?

https://github.com/laravel/framework/blob/9.x/types/Support/Collection.php